<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RESplash</class>
 <widget class="QDialog" name="RESplash">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="forumsButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>114</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Forums</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="wikiButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>114</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Wiki</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="linksLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>62</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Links</string>
   </property>
  </widget>
  <widget class="QPushButton" name="twitterButton">
   <property name="geometry">
    <rect>
     <x>243</x>
     <y>214</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../Reflow.qrc">
     <normaloff>:/twitter_logo.png</normaloff>:/twitter_logo.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>38</width>
     <height>31</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="facebookButton">
   <property name="geometry">
    <rect>
     <x>297</x>
     <y>210</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../Reflow.qrc">
     <normaloff>:/facebook_logo.png</normaloff>:/facebook_logo.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="githubButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>210</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../Reflow.qrc">
     <normaloff>:/github_logo.png</normaloff>:/github_logo.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>38</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="blogButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>114</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Blog</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="licenseLabel">
   <property name="geometry">
    <rect>
     <x>227</x>
     <y>140</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Reflow is Free and Open Source Software, distributed under the CeCILL License.</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="copyrightLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Â© 2011-2013, Gargant Studios, EURL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="websiteButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>114</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Website</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="versionLabel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>0</y>
     <width>62</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>1.7.2</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../Reflow.qrc"/>
 </resources>
 <connections/>
</ui>
